packages/@lwc/integration-karma/test/component/face-callbacks/index.spec.js

```diff
@@ -140,19 +176,23 @@ if (typeof ElementInternals !== 'undefined') {
                     ctor: LightDomFormAssociated,
                 },
             ].forEach(({ name, tagName, ctor }) => {
-                const [face, cecFace] = createFaces(tagName, ctor);
-                it(`${name} does not call face lifecycle methods when upgraded by LWC`, () => {
-                    // TODO [#3929]: Face should log a dev warning when used with synthetic lifecycle
-                    testFaceLifecycleMethodsNotCallable(face);
+                createFaceTests(tagName, ctor, (createFace, scenario) => {
+                    if (scenario === 'lwc.createElement') {
+                        it(`${name} does not call face lifecycle methods when upgraded by LWC`, () => {
+                            testFaceLifecycleMethodsNotCallable(createFace);
+                        });
+                    } else {
+                        // Face throws error message when synthetic shadow is enabled
+                        if (name === 'light DOM' || process.env.NATIVE_SHADOW) {
+                            it(`${name} calls face lifecycle methods when using CustomElementConstructor`, () => {
+                                // CustomElementConstructor is to be upgraded independently of LWC, it will always use native lifecycle
+                                testFaceLifecycleMethodsCallable(createFace);
+                            });
+                        } else {
+                            xit('empty test since jasmine demands every describe() has an it', () => {});
```

Comment:
We could replace the `xit` with the `testFaceLifecycleMethodsNotCallable(createFace);` here instead just to document the behavior of synthetic lifecycle + `CustomElementConstructor` face behavior.